pegasus/sites.v3/code.org/public/js/tabs-section.js

```diff
@@ -25,9 +25,28 @@ tabs.forEach((tab) => {
 });
 
 // Hide the Programming Tools section on code.org/curriculum/computer-vision
-document
-  .querySelector(".resources-tabs-hide-tab button.tab.tools")
-  .classList.add("hide");
-document
-  .querySelector(".resources-tabs-hide-tab details.tools")
-  .classList.add("hide");
+if (document.querySelector(".hide-tab")) {
+  document
+    .querySelector(".hide-tab .resources-tabs button.tab.tools")
+    .classList.add("hide");
+  document
+    .querySelector(".hide-tab .resources-tabs details.tools")
+    .classList.add("hide");
+}
+
+// Hide the desktop or tablet/mobile version in the DOM
+jQuery(document).ready(function ($) {
+  var hideResponsiveTabs = function () {
+    var ww = document.body.clientWidth;
+    if (ww > 1024) {
+      document.querySelector(".show-tablet-and-mobile").remove();
+    } else if (ww <= 1024) {
```

Comment:
Also could each if/else branch include adding in the selector that isn't being removed so that the user can toggle between them and then it wouldn't require a page refresh?

Something like:
`if (ww > 1024) {`
`    remove-tablet-and-mobile();`
`    add-desktop();`
`} else {`
`    remove-desktop();`
`    add-tablet-and-mobile();`
`}`

I think opening the page then switching to a smaller screen size causes both to be hidden currently.

Reply:
Love this solution, thank you! And excellent learning opportunity (ChatGPT helped too haha) 😄

https://github.com/code-dot-org/code-dot-org/assets/9256643/de1f18e5-5785-4c4a-be1b-411ffd75cdc4

Reply:
(cc @dmcavoy this fixes the issue you were having before!)

Reply:
Oh wait, I just double checked this and it's not working as intended — I need to remove the tablet/mobile version from the DOM in order for the Employee Match widget to load and my new solution is just hiding it in the css.